source/configuration.txt

```diff
@@ -1,16 +1,160 @@
+
+   The following sample configuration makes the {+product+} use the 
+   `LINQ3 <{+driver-docs+}reference/driver/crud/linq3/>`__
+   provider.
+
+   .. code-block:: json
+
+      { "DefaultLinqVersion": "V3" }
+
+#. Add the following to the ``.csproj`` for your .NET application within the ``Project`` tag:
+
+   .. code-block:: xml
+
+      <ItemGroup>
+        <AdditionalFiles Include="<path to your {+product-settings-file-constant+} file>" />
+      </ItemGroup>
+
+To learn more about .NET project files, see 
+`Understanding the Project File <https://docs.microsoft.com/en-us/aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/understanding-the-project-file>`__
+from Microsoft.
+
+To learn more about the ``AdditionalFiles`` tag, see 
+:github:`Using Additional Files </dotnet/roslyn/blob/main/docs/analyzers/Using%20Additional%20Files.md>`
+in the Roslyn documentation.
+
+Options
+-------
+
+The following is a description of all configuration options supported
+by the {+product+}:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 35 65
+
+   * - Name
+     - Description
+
+   * - | **DefaultLinqVersion**
+     - | **Type:** string
+       |
+       | **Description:**
+       | The LINQ provider the {+product+} uses.
+         
+       .. tip::
+
+          To learn more about LINQ, see the
+          :ref:`Analyze Your Code <analyzer-analyze-linq>`
+          page.
+
+       | **Default**: ``"V2"``
+       | **Accepted Values**: ``"V2"`` or ``"V3"``
+
+   * - | **OutputDriverVersion**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Specifies if the {+product+} includes your {+driver-short+} version in diagnostic messages.
+       |
+       | **Default**: ``false``
+       | **Accepted Values**: ``true`` or ``false``
+
+   * - | **OutputInternalExceptions**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Specifies if the {+product+} includes internal exceptions in diagnostic messages.
+       |
+       | **Default**: ``false``
+       | **Accepted Values**: ``true`` or ``false``
+
+   * - | **OutputInternalLogsToFile**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Specifies if the {+product+} writes its internal logs to a file.
+
+       .. include:: includes/logging-admonition.rst       
+
+       | **Default**: ``false``
+       | **Accepted Values**: ``true`` or ``false``
+
+   * - | **LogFileName**
+     - | **Type:** string
+       |
+       | **Description:**
+       | Specifies the path to which the {+product+} writes its internal logs.
+       
+       .. include:: includes/logging-admonition.rst       
+
+       | **Default**: ``""``
+       | **Accepted Values**: A valid file path
+   * - | **SendTelemetry**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | Specifies if the {+product+} sends anonymized information to MongoDB (the company).
```

Comment:
question: Why add "(the company)"? I can't find any other usage like this anywhere in the documentation. Could this be worded to avoid the confusion I'm guessing you're trying to avoid?

```suggestion
       | Specifies if the {+product+} sends anonymized information for continued improvement.
```

Reply:
The only other telemetry statement I was aware of was this [mongosh page](https://docs.mongodb.com/mongodb-shell/telemetry/) page Max directed me to, and I believe a shortcoming of this page is it does not explicitly answer the question "Who is getting my data?" instead only answering "Why are you collecting my data?" (Answer: "to improve products").

I think "(the company)" should be changed. I think "MongoDB Inc." might be a good alternative (this is used in the [Privacy Policy](https://www.mongodb.com/legal/privacy-policy)).

Reply:
That works.